src/Components/script.js

```diff
@@ -0,0 +1,7 @@
+document
+  .getElementById("cameraFileInput")
```

Comment:
lets call this file somehting other than script.js something more relevent to what it is doing